README.md

```diff
@@ -156,6 +159,104 @@ successful. You can disable this feature with `useCache` router option.
 
 **Be careful, this cache is global and cannot be disabled for special pages.**
 
+### Transitions
+
+To manage transitions, you can set `data-transition` attribute to an animation name on links.
+
+```html
+<a href="/contact" data-transition="fade">Contact</a>
+```
+
+Then, create a **TransitionFactory** class and pass it to the **Router** instance.  
+In this class, implement `getTransition (previousState, state, direction = null)`. 
+This method is called on each transition and give you access to state informations :  
+
+- `previousState` and `state`
+	- **transitionName** : `data-transition` attributes of the clicked link
+	- **context** : equal to `"history"`, `"link"` or `"nav"` 
+- `direction` : equal to `"back"` or `"forward"` on navigator buttons click (only when `context` equals to `"history"`)
+
+Example: 
+```javascript
+// src/transition-factory.js
+
+import DefaultTransition from './transitions/default-transition';
+import FadeTransition from './transitions/fade-transition';
+  
+export default class TransitionFactory {
+    getTransition(previousState, state, direction = null) {
+        switch (state.transitionName) {
+            case 'fade':
+                return new FadeTransition();
+                break;
+            default:
+                return new DefaultTransition();
+                break;
+        }
+    }
+}
+```
+
+To create a new transition you need to create a new class extending `AbstractTransition`. Implement `start()` method and use Promises to manages your animations.  
+
```

Comment:
Explain that `this.newContainerLoading` promise is **mandatory** to actually load new page.